### ISSUE TITLE
Bug: Disabled buttons look prominent

### ISSUE THREAD

**Issue by tedgoas-dialpad** [dialpad/dialtone#143]
It's a little weird how prominent the disabled buttons appear compared to the non-disabled ones. Here's an example:
<img width="291" alt="Screen Shot 2020-11-18 at 10 58 44 AM" src="https://user-images.githubusercontent.com/65190890/99554414-2a325000-298d-11eb-8cf9-4f66ca064624.png">

---

Can we revisit the style of our disabled buttons? Maybe we need different `disabled` styles for muted vs. outlined vs. primary buttons styles.

**Comment by abasolo-dialpad** [comment]
I agree, I just bumped against this same problem. It's especially odd when having two buttons stacked on top of one another in a potential scenario like this: 
![image](https://user-images.githubusercontent.com/64873219/99733099-7ce73700-2a75-11eb-8f63-647503c63701.png)

I think outlined/primary could share a disabled style but muted and default should have a separate one, perhaps with no fill (but we can explore what that could look like).

**Comment by abasolo-dialpad** [comment]
Adding on, looked into Material and looks like they recommend no fill for muted/outlined and filled only for a primary button
![image](https://user-images.githubusercontent.com/64873219/99735420-70fd7400-2a79-11eb-8325-2b41910b3e10.png)
![image](https://user-images.githubusercontent.com/64873219/99735441-778beb80-2a79-11eb-8537-2efc31c0c6c1.png)
![image](https://user-images.githubusercontent.com/64873219/99735450-7ce93600-2a79-11eb-9d34-d61daf2cc554.png)
